src/marketplace/RemoveASaleView.js

```diff
@@ -0,0 +1,86 @@
+//@flow strict
+import * as React from 'react';
+import { ToastContainer } from 'react-toastify';
+import { toTitleCase } from '../globalComponents/Utility.js';
+import SaleObject from '../globalComponents/typesAndClasses/SaleObject.js';
+import getUserAPICall from '../globalComponents/apiCalls/getUserAPICall.js';
+import { getUsersCurrentSales, removeASale } from '../globalComponents/apiCalls/marketPlaceAPIConnections.js';
+
+type Props = {|
+|}; 
+
+type State = {|
+	userId: string,
+	itemsUserHasForSale: Array<SaleObject>
+|};
+
+class RemoveASaleView extends React.Component<Props, State> {
+
+	constructor() {
+		super();
+		this.state={
+			userId: '',
+			itemsUserHasForSale: [],
+		}
+	}
+
+	componentDidMount() {
+		this.getUserID();
+	}
+
+
+	//this gets the user id
+	getUserID() : void {
```

Comment:
If you copied this from somewhere, that should be an indication that something needs to be done differently. If this code is unique, then why is this its own method if it is only called in one other method that does nothing but call this method?